Ownership/AC - South/Dover.txt

```diff
@@ -1,14 +1,39 @@
-SECTOR:London AC Dover:0:66000
+SECTOR:London AC Dover:19500:66000
+;Main Area over DVR
 OWNER:LD:LS:LWS:LSC:L
 ALTOWNER:Observing LON_S_CTR:LD:O:LM:LS:LWS:LSC:L
 ALTOWNER:Observing LON_D_CTR:O:LM:LD:LS:LWS:LSC:L
 ALTOWNER:Observing London FIR:O:LM:LD:LS:LWS:LSC:L
-BORDER:LCS3:LCSD1:LCSD2:LTC5:LESD1:LED1:LED2:LED3:LED4:LDBU1:LDFF1:LDFF2:LDM2:LDM1:LDM16:LTC10:LTC13:LTC15:LTC18:LSD1
+BORDER:LTC28:LED1:LED2:LTC12:LDFF2:LDM3:LDM4:LTC11:LTC13
 GUEST:TCSE:*:EGLC
 GUEST:TCSE:*:EGMC
 GUEST:TCS:*:EGLC
 GUEST:TCS:*:EGMC
 
+;Capital Block (High)
+SECTOR:London AC Dover 2:21500:660000
+OWNER:LD:LS:LWS:LSC:LCS3
```

Comment:
ownership shouldnt include LCS3 (not a defined position)

Reply:
That's come from the line beneath - will amend.